serhii.trokhymets/homework_6/document.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+
+class Document
+  attr_accessor :doc
+
+  def initialize(path)
+    @path = path
+  end
+
+  def parse
+    content = File.read(@path)
+    @doc = Nokogiri::HTML(content)
```

Comment:
You've declared an attr_accesor in line 6, please use setter here or remove the accessor

Reply:
attr_reader :doc ?
I'm at work now, can't fix it by 18:30